frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/EnvironmentHeader.tsx

```diff
@@ -56,7 +54,12 @@ export const EnvironmentHeader: FC<EnvironmentHeaderProps> = ({
         <StyledAccordionSummary
             {...props}
             expandIcon={
-                expandable ? <ExpandMore titleAccess='Toggle' /> : <></>
+                <ExpandMore
+                    titleAccess='Toggle'
```

Comment:
I'd put the env name in here, too; e.g. `Toggle ${environment} information` or something.